packages/dds/tree/api-report/tree.beta.api.md

```diff
@@ -252,6 +253,22 @@ type ObjectFromSchemaRecordUnsafe<T extends Unenforced<RestrictiveReadonlyRecord
 // @public
 export type Off = () => void;
 
+// @public @sealed
+interface ReadonlyMapInlined<K, T extends Unenforced<ImplicitAllowedTypes>> {
```

Comment:
Should this be a part of the internal types namespace export?

Reply:
Ah, or was that just replaced with `@system`?

Reply:
It is part of internalTypes (which is a module not a namespace currently). I redundantly put system and sealed on it for extra clarity.

Reply:
Note in this APi report, this definition is not exported: its exported from the namespace (thats how API extractor handles namespaces)